server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('./dist'));
+
```

Comment:
Сейчас при запуске приложения локально через npm run start можно получить страницу только запрашивая напрямую index.html
Стоит добавить
```suggestion
app.use('*', (_req, res) => {
  res.sendFile(path.join(__dirname, 'dist/index.html'));
});
```